### PR TITLE
Split testCMCSingleMuscle into two to speed it up

### PR DESCRIPTION
**Pull request by klshrinidhi**
Similar to #1131 and #1127.

### CODE REVIEW

**Comment by tkuchida** [comment]
LGTM :+1:

**Comment by tkuchida** [comment]
(The test failure is the familiar [testContactGeometry timeout on clang in Debug mode](https://travis-ci.org/opensim-org/opensim-core/jobs/145983863#L1961).)

File: Applications/CMC/test/testCMCSingleMuscle_Thelen.cpp [hunk 1]
```diff
@@ -35,15 +35,15 @@ void testSingleMuscle() {
...
+    // forward.setResultsDir("block_hanging_from_muscle_ForwardResults_Thelen");
     forward.run();
 
     CMCTool cmc("block_hanging_from_muscle_Setup_CMC.xml");
-    cmc.setResultsDir("block_hanging_from_muscle_ResultsCMC_Thelen");
+    // cmc.setResultsDir("block_hanging_from_muscle_ResultsCMC_Thelen");
     cmc.run();
 
-    Storage fwd_result("block_hanging_from_muscle_ForwardResults_Thelen/block_hanging_from_muscle_states.sto");
-    Storage cmc_result("block_hanging_from_muscle_ResultsCMC_Thelen/block_hanging_from_muscle_states.sto");
```
Comment (chrisdembia) [inline Applications/CMC/test/testCMCSingleMuscle_Thelen.cpp]: What's the issue here? How come changing the results directory caused an issue? Seems preferable.

**Comment by chrisdembia** [comment]
These changes look correct. However I would feel more comfortable if the setup files `block_hanging_from_muscle_Setup_CMC.xml` and `block_hanging_from_muscle_Setup_Forward.xml` had bogus values for `desired_kinematics_file`, `results_directory` to absolutely ensure that the tests are not using incorrect inputs.

File: Applications/CMC/test/testCMCSingleMuscle_Thelen.cpp (hunk 1 above)
Comment (chrisdembia) [inline Applications/CMC/test/testCMCSingleMuscle_Thelen.cpp]: I understand now; you can ignore this comment.

**Comment by aseth1** [comment]
Closing because #1146 makes splitting of the test unnecessary.

**Pull request closed without merging**